docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md

```diff
@@ -318,14 +318,57 @@ npm:
 
 ---
 
-### `registry`
```

Comment:
What about keeping `registry` and just mentioning it's deprecated and using `registries` instead?